Jellyfin.Data/Entities/User.cs

```diff
@@ -413,6 +413,52 @@ public string[] GetPreference(PreferenceKind preference)
             return Equals(val, string.Empty) ? Array.Empty<string>() : val.Split(Delimiter);
         }
 
+        /// <summary>
+        /// Gets the user's preferences for the given preference kind.
+        /// </summary>
+        /// <param name="preference">The preference kind.</param>
+        /// <typeparam name="T">Type of preference.</typeparam>
+        /// <returns>A {T} array containing the user's preference.</returns>
+        public T[] GetPreference<T>(PreferenceKind preference)
```

Comment:
I would rename it to something like `GetPreferenceValues` or `GetAllPreferenceValues`. It's not immediately apparent that it's a list of values just by the name. A more general approach would be to have the signature as `public T GetPreference<T>`, but that complicates it a bit.